events/app.py

```diff
@@ -67,8 +67,6 @@ def add_event():
             'author': request.form['author_id'],
             'event_time': request.form['event_time']
         }
-        # TODO(cmei4444): Athenticate user and verify that user has event
```

Comment:
Presumably because we changed the pattern of where authorization is requested in #62?

Reply:
Correct, see https://github.com/knative-portability/large-events/pull/68/commits/c679c15d4dad5b8f38ee1e9bb7cb77858e5edf5e